rules/windows/driver_load/driver_load_vuln_drivers.yml

```diff
@@ -864,8 +864,6 @@ detection:
             - 'DB90E554AD249C2BD888282ECF7D8DA4D1538DD364129A3327B54F8242DD5653'
             - 'E61A54F6D3869B43C4ECEAC3016DF73DF67CCE03878C5A6167166601C5D3F028'
             - '3871E16758A1778907667F78589359734F7F62F9DC953EC558946DCDBE6951E3'
-            - 'DED2927F9A4E64EEFD09D0CABA78E94F309E3A6292841AE81D5528CAB109F95D'
-            - '0296E2CE999E67C76352613A718E11516FE1B0EFC3FFDB8918FC999DD76A73A5'
```

Comment:
those are removed because they are duplicates. Can you sort the hashes? So this can be seen easier in the future